app/models/rule_result.rb

```diff
@@ -34,7 +34,7 @@ class RuleResult < ApplicationRecord
   scope :failed, -> { where(result: FAILED) }
   scope :for_system, ->(host_id) { where(host_id: host_id) }
   scope :for_policy, ->(policy_id) { joins(:profile).where(profiles: ::Profile.in_policy(policy_id)) }
-  scope :latest, ->(policy_id) { for_policy(policy_id).joins(:test_result).joins(::TestResult.with_latest) }
+  scope :latest, ->(policy_id) { for_policy(policy_id).joins(:test_result).merge(TestResult.latest) }
```

Comment:
This is a residual change, the time I wrote this, I did not know about the `merge` :grin: